lib/oeqa/selftest/cases/updater_qemux86_64.py

```diff
@@ -150,7 +150,7 @@ def test_manual_run_mode_once(self):
 
         stdout, stderr, retcode = self.qemu_command('aktualizr once')
 
-        stdout, stderr, retcode = self.qemu_command('aktualizr-info')
+        stdout, stderr, retcode = self.qemu_command('aktualizr-info --wait-until-provisioned')
```

Comment:
This particular function actually shouldn't have that, since the calls are made sequentially here. If `aktualizr once` didn't succeed in provisioning the device, something went wrong and there's nothing more to wait for.